### PR TITLE
Profile overlay section display not always correct

### LINKED ISSUE #8704: Profile overlay section display not always correct
**Describe the bug:** 
Upper bar (navigation bar, i guess) in in-game user profile shows that i'm at "Kudosu!" section, while i'm still at "Beatmaps". I scrolled about 3 screens down from start of "Beatmaps" section, and it started to occur.
**Screenshots or videos showing encountered issue:** 
osu_2020-04-10_08-04-16
**osu!lazer version:** 
2020.408.1
**Logs:**
Don't think it needs any

### PR DESCRIPTION
**Pull request by ghost**
With this commit, the selected section changes as soon as the screen top border enters the new section, instead of choosing the closer section.

### CODE REVIEW

**Comment by bdach** [comment]
Please do not close and reopen pull requests after a CI failure, and also please do not edit code via the web interface. This makes me have very little (if not zero) confidence in the changes introduced here.

**Comment by Fukashi13** [comment]
I'm sorry, this is my first contribution to a Git project. I pasted the changes from web because the folder was a bit in a mess and I wanted to get sure that only this file gets commited

**Comment by bdach** [comment]
In that case I would recommend that you familiarise yourself with git and pull request workflow (staging changes, resetting files, committing, pushing) before contributing further. Otherwise this is going to be too much of a pain to manage.

**Comment by Fukashi13** [comment]
Ok, won't happen again, thanks for the tipps

File: osu.Game/Graphics/Containers/SectionsContainer.cs [hunk 1]
```diff
@@ -190,14 +190,13 @@ protected override void UpdateAfterChildren()
                 headerBackgroundContainer.Y = ExpandableHeader?.Y ?? 0;
 
                 T bestMatch = null;
-                float minDiff = float.MaxValue;
-                float scrollOffset = FixedHeader?.LayoutSize.Y ?? 0;
```
Comment (peppy) [inline osu.Game/Graphics/Containers/SectionsContainer.cs:194]: why was this moved out of a constant?

**Comment by bdach** [comment]
@Fukashi13 please do not force-push either until there's a very good reason to do so. Force-pushing does not work well with CI and should only be used in extraordinary situations.

**Comment by peppy** [comment]
also don't PR from master.

**Comment by Fukashi13** [comment]
> @Fukashi13 please do not force-push either until there's a very good reason to do so. Force-pushing does not work well with CI and should only be used in extraordinary situations.

I squashed the commits made on web

File: osu.Game/Graphics/Containers/SectionsContainer.cs (hunk 1 above)
Comment (Fukashi13) [reply osu.Game/Graphics/Containers/SectionsContainer.cs:194]: Is it ok now?
